Utilities/CICD/Azure-DevOps/README.md

```diff
@@ -44,3 +44,13 @@ packagePath=<package file path, e.g., './Calculator.zip'>
 
 e.g.,CreateOrUpdatePackage.ps1 -applicationName <application name in Test Base> -packageVersion <application version in Test Base> -packagePath=<package file path>
 ```
+### For multiple packages
```

Comment:
### Create multiple packages
To create multiple packages, you can create multiple CICD workflows, and update the variables in [config.json](../../../Sample-App-Src/Calculator/scripts/config.json) for each workflow.